views/reset-password.ejs

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Barefoot Nomad - Reset confirmPassword</title>
+</head>
+<body>
+  <form method="POST">
+    <label for="password">New Password</label>
+    <input type="password" name="password" placeholder="Enter a new password">
+    <label for="confirmPassword">Confirm Password</label>
+    <input type="password" name="confirmPassword" placeholder="Repeat the new password">
+    <button type="submit">Reset Password</button>
+  </form>
+</body>
+</html>
```

Comment:
Password reset functionalities can be easily tested on the backend.
This page will be designed on the Frontend.
Remove it

Reply:
@niyojuste is it for just demonstration ? if so you can leave it